msteams-platform/m365-apps/prerequisites.md

```diff
@@ -44,44 +43,24 @@ You also need to enable sideloading for your tenant:
 
  1. Turn on **Upload custom apps**, and select **Save**.
 
-## Enroll your developer tenant for Microsoft 365 Targeted Releases
```

Comment:
Let's keep this section, but replace the [!IMPORTANT] note (which I don't believe is needed anymore) with a non-urgent [!NOTE] that this section is optional and only required for any features in preview, as listed in [Teams apps across Microsoft 365](https://learn.microsoft.com/microsoftteams/platform/m365-apps/overview) app element support table.